llamaindex_rag.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<p>Please add a little description to each sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>